### PR TITLE
correct USB PIDs for several boards

### LINKED ISSUE #1891: Some boards unintentionally sharing USB_PID values
I was reviewing USB PID allocations, as you do, and noted that the PyPortal and GrandCentral M4 use the _same_ one, **0x8032**. @tannewt confirmed in discord that this case of sharing is incorrect.

Here's a summary of all of them in PID order generated from the `ports/atmel-samd/board` directory:

```
$ date ; grep USB_PID $(grep 'USB_VID.*=.*0x239[aA]' -l */*.mk) | awk -F: '{ print $2, $1; }' | sort -k 3
Thu 16 May 10:58:36 BST 2019
USB_PID = 0x8012 itsybitsy_m0_express/mpconfigboard.mk
USB_PID = 0x8014 metro_m0_express/mpconfigboard.mk
USB_PID = 0x8015 feather_m0_adalogger/mpconfigboard.mk
USB_PID = 0x8015 feather_m0_basic/mpconfigboard.mk
USB_PID = 0x8015 feather_m0_rfm69/mpconfigboard.mk
USB_PID = 0x8015 feather_m0_rfm9x/mpconfigboard.mk
USB_PID = 0x8019 circuitplayground_express_crickit/mpconfigboard.mk
USB_PID = 0x8019 circuitplayground_express/mpconfigboard.mk
USB_PID = 0x801D gemma_m0/mpconfigboard.mk
USB_PID = 0x801D pewpew10/mpconfigboard.mk
USB_PID = 0x801F trinket_m0_haxpress/mpconfigboard.mk
USB_PID = 0x801F trinket_m0/mpconfigboard.mk
USB_PID = 0x801F ugame10/mpconfigboard.mk
USB_PID = 0x8021 cp32-m4/mpconfigboard.mk
USB_PID = 0x8021 metro_m4_express/mpconfigboard.mk
USB_PID = 0x8023 feather_m0_express_crickit/mpconfigboard.mk
USB_PID = 0x8023 feather_m0_express/mpconfigboard.mk
USB_PID = 0x8023 feather_m0_supersized/mpconfigboard.mk
USB_PID = 0x8023 feather_radiofruit_zigbee/mpconfigboard.mk
USB_PID = 0x8026 feather_m4_express/mpconfigboard.mk
USB_PID = 0x8028 pirkey_m0/mpconfigboard.mk
USB_PID = 0x802C itsybitsy_m4_express/mpconfigboard.mk
USB_PID = 0x802C pybadge/mpconfigboard.mk
USB_PID = 0x802E pygamer/mpconfigboard.mk
USB_PID = 0x8030 trellis_m4_express/mpconfigboard.mk
USB_PID = 0x8032 grandcentral_m4_express/mpconfigboard.mk
USB_PID = 0x8032 pyportal/mpconfigboard.mk
USB_PID = 0x8035 arduino_mkrzero/mpconfigboard.mk
USB_PID = 0x8038 metro_m4_airlift_lite/mpconfigboard.mk
USB_PID = 0xD1ED hallowing_m0_express/mpconfigboard.mk
```

Perhaps the `itsybitsy_m4_express` sharing with `pybadge` is incorrect too? I don't know what a `pewpew10` is, is it a variant of a Gemma M0, they are sharing too?

`hallowing_m0_express` is an anomaly with its number clearly from a different range.

### PR DESCRIPTION
**Pull request by dhalbert**
Fixes #1891. Checked against master internal list. @ladyada tagging you for info.

### CODE REVIEW

**Comment by dhalbert** [comment]
Oops, I didn't mean to merge! It didn't start a check. Please review anyway. I'll fix the branch settings to require approval.

File: ports/atmel-samd/boards/arduino_mkrzero/mpconfigboard.mk [hunk 1]
```diff
@@ -1,6 +1,6 @@
 LD_FILE = boards/samd21x18-bootloader.ld
 USB_VID = 0x239A
-USB_PID = 0x8035
+USB_PID = 0x8050
```
Comment (tannewt) [inline ports/atmel-samd/boards/arduino_mkrzero/mpconfigboard.mk]: why does the arduino use an adafruit vid?

Comment (dhalbert) [reply ports/atmel-samd/boards/arduino_mkrzero/mpconfigboard.mk]: We’re replacing the regular bootloader and don’t have multiple pids available from Arduino. This was assigned a long time ago but had errors.

Comment (tannewt) [reply ports/atmel-samd/boards/arduino_mkrzero/mpconfigboard.mk]: kk

**Pull request merged**